cli.js

```diff
@@ -19,16 +19,16 @@ program
   .description(
     "Translate a text and put the result on the files in the output directory",
   )
-  .argument("<text>", "Text to translate")
-  .argument("<source-language>", "Source language of the string")
-  .argument("<name-of-translation>", "Name of your translation")
+  .option("-t, --text <string>", "Text to translate")
+  .option("-f, --from <string>", "Source language of the string")
+  .option("-n, --name <string>", "Name of the translation")
   .option(
     "-e, --engine <string>",
-    "Engine to use for the translation. In case you don't define it will be use by default all the translation engines that are free and doesn't requires API Key.",
+    "[OPTIONAL]. Engine to use for the translation. In case you don't define it will be use by default all the translation engines that are free and doesn't requires API Key.",
```

Comment:
Here is a little bit confusing which are the default engines, I propose to define explicitly which ones are used or in which file to check the current defaults

Reply:
You're right we need improve tool commands descriptions so users doesn't needs check readme. Will create a task for that. Thanks!